packages/modules/cra-template-itwin-viewer/template/.env

```diff
@@ -1,9 +1,9 @@
 # ---- Authorization Client Settings ----
-REACT_APP_AUTH_CLIENT_CLIENT_ID=imodeljs-spa-samples-2686
-REACT_APP_AUTH_CLIENT_REDIRECT_URI=http://localhost:3000/signin-callback
-REACT_APP_AUTH_CLIENT_LOGOUT_URI=http://localhost:3000/logout
-REACT_APP_AUTH_CLIENT_SCOPES ="openid email profile organization imodelhub context-registry-service:read-only product-settings-service general-purpose-imodeljs-backend imodeljs-router"
+IMJS_AUTH_CLIENT_CLIENT_ID=imodeljs-spa-samples-2686
+IMJS_AUTH_CLIENT_REDIRECT_URI=http://localhost:3000/signin-callback
+IMJS_AUTH_CLIENT_LOGOUT_URI=http://localhost:3000/logout
+IMJS_AUTH_CLIENT_SCOPES ="openid email profile organization imodelhub context-registry-service:read-only product-settings-service general-purpose-imodeljs-backend imodeljs-router"
```

Comment:
we should probably add the scope `urlps-third-party`

Reply:
what is that scope for?

Reply:
added